### PR TITLE
Document dashboardInitialized event and clean up existing tile docs

### PR DESCRIPTION
**Pull request by blackbaud-johnly**
Addresses https://github.com/blackbaud/skyux2/issues/1939.

### CODE REVIEW

File: src/app/components/demo-components.service.ts [hunk 1]
```diff
@@ -422,7 +422,7 @@ export class SkyDemoComponentsService {
         name: 'Tile',
         icon: 'th-large',
         // tslint:disable-next-line
-        summary: `The tile directive provides a container that is the building block for pages and forms.`,
+        summary: `The tile directive component creates a collapsible container that is a building block for pages and forms.`,
```
Comment (Blackbaud-TrevorBurch) [inline src/app/components/demo-components.service.ts]: The tile would be either a `component` or a `directive` but not both.  It appears the old version had `directive` a lot and I like the change to `component`. We should probably just remove the word `directive` here too.

File: src/app/components/tile/index.html [hunk 2]
```diff
@@ -68,25 +68,33 @@
...
+      A <stache-code>SkyTileDashboardConfigLayout</stache-code> object that describes the tile dashboard's layout. The <stache-code>SkyTileDashboardConfigLayout</stache-code> object contains a <stache-code>singleColumn</stache-code> property of type <stache-code>SkyTileDashboardConfigLayoutColumn</stache-code> and a <stache-code>multiColumn</stache-code> property that is an array of type <stache-code> SkyTileDashboardConfigLayoutColumn</stache-code>. The <stache-code>singleColumn</stache-code> layout displays on small screens, while the <stache-code>multiColumn</stache-code> layout displays on larger screens. The <stache-code>SkyTileDashboardConfigLayoutColumn</stache-code> object contains an array of <stache-code>SkyTileDashboardConfigLayoutTile</stache-code> objects that indicates the order of tiles. It includes the following properties:
```
Comment (Blackbaud-TrevorBurch) [inline src/app/components/tile/index.html]: There is an extra space in this portion of the text talking about the `multiColumn` property.

```
 is an array of type <stache-code> SkyTileDashboardConfigLayoutColumn</stache-code>
```

File: src/app/components/tile/index.html [hunk 3]
```diff
@@ -28,38 +28,38 @@
 
   <sky-demo-page-properties sectionHeading="Tile events">
     <sky-demo-page-property
-      propertyName="settingsClick"
+      propertyName="isCollapsedChange"
     >
-    Event emitted when users click the settings button on the tile. The settings button only appears on the tile if the <stache-code>showSettings</stache-code> is set to <stache-code>true</stache-code>.
+    Fires when the tile's collapsed state changes. Returns <stache-code>true</stache-code> when the tile is collapsed and <stache-code>false</stache-code> whent it is not.
```
Comment (Blackbaud-AlexKingman) [inline src/app/components/tile/index.html]: Typo? `whent it is not` should be `when it is not`.

File: src/app/components/demo-components.service.ts (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply src/app/components/demo-components.service.ts]: Agreed. No need for `directive`.

Comment (blackbaud-johnly) [reply src/app/components/demo-components.service.ts]: Thanks for catching that. I removed the old instances of `directive` after confirming the change with Steve but apparently missed this one. Thanks!

**Comment by blackbaud-johnly** [comment]
@Blackbaud-AlexKingman and @Blackbaud-TrevorBurch, I updated to address the feedback. Can one of you please approve the PR so that I can merge it after the checks pass? Thanks!

File: src/app/components/tile/index.html (hunk 2 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/components/tile/index.html]: @blackbaud-johnly the extra space is still here.

**Comment by Blackbaud-TrevorBurch** [comment]
@blackbaud-johnly one small comment was missed. I tagged you in it above (it is hidden due to the original commit being "outdated").

File: src/app/components/tile/index.html (hunk 2 above)
Comment (blackbaud-johnly) [reply src/app/components/tile/index.html]: Weird. I know I deleted it. Not sure how I messed that up, but I'll fix it now.

Comment (blackbaud-johnly) [reply src/app/components/tile/index.html]: Actually, I just opened the file again to remove the space, and I don't see the extra space anymore.

**Comment by blackbaud-johnly** [comment]
@Blackbaud-TrevorBurch, are you still seeing the extra space? I double-checked, and I'm not seeing it.

**Comment by Blackbaud-TrevorBurch** [comment]
@blackbaud-johnly nope. Not sure why I was still seeing it yesterday but it is gone now. LGTM!

**Pull request merged**